Test/TestCase/SearchManagerTest.php

```diff
@@ -158,12 +159,49 @@ public function testRemove()
         $this->assertNull($view);
     }
 
+    public function testRemoveAll()
```

Comment:
Oh yeah, that one only does a single entity. We need some test cases for multiple entities and also a timeseries index.